src/webgpu/web_platform/copyToTexture/ImageBitmap.spec.ts

```diff
@@ -36,6 +36,8 @@ enum Color {
 }
 // Cache for generated pixels.
 const generatedPixelCache: Map<GPUTextureFormat, Map<Color, Uint8Array>> = new Map();
+type TransparentOp = 'premultiply' | 'none' | 'non-transparent';
```

Comment:
nit: Comment saying these two types correspond to `premultiplyAlpha` and `imageOrientation` in `ImageBitmapOptions`.

Reply:
Done.